include/SPTPersistentCache/SPTPersistentCacheOptions.h

```diff
@@ -32,6 +32,22 @@ NS_ASSUME_NONNULL_BEGIN
  */
 typedef void (^SPTPersistentCacheDebugCallback)(NSString *message);
 
+typedef NS_ENUM(NSUInteger, SPTPersistentCacheDebugTimingType) {
+    SPTPersistentCacheDebugTimingTypeQueued = 0,
+    SPTPersistentCacheDebugTimingTypeStarting = 1,
+    SPTPersistentCacheDebugTimingTypeFinished = 2
+};
+
+typedef NS_ENUM(NSUInteger, SPTPersistentCacheDebugMethodType) {
+    SPTPersistentCacheDebugMethodTypeStore = 0,
```

Comment:
Since the raw value of the enum value isn’t used or relied up we could remove those. Should make maintenance easier.